src/mlpack/core/optimizers/sgd/sgd.hpp

```diff
@@ -135,6 +136,9 @@ class SGD
   //! Modify whether or not the individual functions are shuffled.
   bool& Shuffle() { return shuffle; }
 
+  //! Get the update policy. 
+  UpdatePolicy GetUpdatePolicy() const { return updatePolicy; }
```

Comment:
yes, but please only rename the type not the parameter so:

```
const UpdatePolicy updatePolicy = UpdatePolicy());
```

would become 

```
const UpdatePolicyType updatePolicy = UpdatePolicyType());
```

and not

```
const UpdatePolicyType updatePolicyType = UpdatePolicyType());
```

and also don't forget to add:

```
UpdatePolicyType& UpdatePolicy() { return updatePolicy; }
```

to the sgd class.